ENOENT: no such file or directory, chmod '...\node_modules\vue-native-scripts\src\bin\vue-native-script.js'

Trying to Install last version 0.1.0
`npm install vue-native-scripts@0.1.0`

but get following error in console:
```
npm ERR! path C:\sample\node_modules\vue-native-scripts\src\bin\vue-native-script.js
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod 'C:\sample\node_modules\vue-native-scripts\src\bin\vue-native-script.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```
and package did not installed

I suppose, this is cause in package.json wrong path to vue-native-script.js file:
```
"bin": {
    "vue-native-scripts": "./src/bin/vue-native-script.js"
  },
```
It should be:
```
"bin": {
    "vue-native-scripts": "./bin/vue-native-script.js"
  },
```

--- a/packages/vue-native-scripts/package.json
+++ b/packages/vue-native-scripts/package.json
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {},
   "bin": {
-    "vue-native-scripts": "./src/bin/vue-native-script.js"
+    "vue-native-scripts": "./bin/vue-native-script.js"
   },
   "author": "Geekyants (https://geekyants.io/)",
   "license": "MIT",

